vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -56,29 +59,28 @@ type internal FSharpCheckerProvider
             // When the F# background builder refreshes the background semantic build context for a file,
             // we request Roslyn to reanalyze that individual file.
             checker.BeforeBackgroundFileCheck.Add(fun (fileName, extraProjectInfo) ->  
-               async {
-                try 
-                    match extraProjectInfo with 
-                    | Some (:? Workspace as workspace) -> 
-                        let solution = workspace.CurrentSolution
-                        let documentIds = solution.GetDocumentIdsWithFilePath(fileName)
-                        if not documentIds.IsEmpty then 
-                            let docuentIdsFiltered = documentIds |> Seq.filter workspace.IsDocumentOpen |> Seq.toArray
-                            for documentId in docuentIdsFiltered do
-                                Trace.TraceInformation("{0:n3} Requesting Roslyn reanalysis of {1}", DateTime.Now.TimeOfDay.TotalSeconds, documentId)
-                            if docuentIdsFiltered.Length > 0 then 
-                                analyzerService.Reanalyze(workspace,documentIds=docuentIdsFiltered)
-                    | _ -> ()
-                with ex -> 
-                    Assert.Exception(ex)
-                } |> Async.StartImmediate
+                async {
```

Comment:
There are still unrelated whitespace changes here - please remove them to minimize the diff

Reply:
hmm I thought I got them all.

Reply:
@KevinRansom I'd like to recommend [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) if you don't already have it installed.  Removes all trailing whitespace on save, very handy.